examples/complete/main.tf

```diff
@@ -0,0 +1,68 @@
+module "complete" {
+  source             = "../../"
+  name               = "complete-example"
+  hash_key           = "id"
+  range_key          = "title"
+  billing_mode       = "PROVISIONED"
+  read_capacity      = 3 ## 3 strongly consistent read per second, or 6 eventually consistent reads per second
+  write_capacity     = 4 ## 4 writes per second, for an item up to 1 KB in size.
+  enable_autoscaling = true
+  create_sse_kms_key = true
+
+  ## For both read and write, these values are set like so to allow faster scale out and slow scale down
+  ## Max and min_capacities: provisioned capacity that your table will allow. Cannot scale above/below these values.
+  autoscaling_read = {
+    scale_in_cooldown  = 180
+    scale_out_cooldown = 60
+    target_value       = 50
```

Comment:
Also match min on autoscaling- with the setting of the table, to avoid inconsistencies use the same variable for autoscaling and table some how in the module itself, for example when it has autoscaling set to true the min read capacity var is the same as the read capacity

Reply:
@hugoalmeida264 Thank you for the insight. I have now implemented it